### ISSUE TITLE
Indentation Error 

### ISSUE THREAD

**Issue by sourcecode369** [lucko515/chatbot#5]

**Comment by suvidh** [comment]
I get this indentation error:-
![image](https://user-images.githubusercontent.com/19871090/53618229-71ecd300-3c10-11e9-9296-0fe545a4b3af.png)

**Comment by sourcecode369** [comment]
Yes Ive got the same error. I guess there is some problem with this implementation. So I had to reimplement it all on my own.

**Comment by vishal2612200** [comment]
This error is generated due to unwanted starting space in some comments. Reduce the starting space of all comments to 4 spaces. Your problem will get solved.
`
def encoder(inputs, rnn_size, number_of_layers, encoder_seq_len, keep_probs, encoder_embed_size, encoder_vocab_size):

    '''
		Used to define encoder of the seq2seq model (The encoder is made of simple dynamic RNN network).

		Inputs:
			inputs -
			rnn_siz - number of units in the RNN layer
			number_of_layer - number of RNN layers that the model uses
			encoder_seq_len - vector of lengths (got from placeholder)
			keep_probs - dropout rate
			encoder_embed_size - size of embedding vector for encoder part
			encoder_vocab_size - number of different words that the model uses in a vocabulary
		
		Outputs:
			encoder_outputs -
			encoder_states - internal states from the RNN layer(s)

    '''
    
    def cell(units, rate):
        layer = tf.contrib.rnn.BasicLSTMCell(units)
        return tf.contrib.rnn.DropoutWrapper(layer,rate)
    
    encoder_cell = tf.contrib.rnn.MultiRNNCell([cell(rnn_size, keep_probs) for _ in range(number_of_layers)])
     
    encoder_embedings = tf.contrib.layers.embed_sequence(inputs, encoder_vocab_size, encoder_embed_size) #used to create embeding layer for the encoder
    
    encoder_outputs, encoder_states = tf.nn.dynamic_rnn(encoder_cell,encoder_embedings, encoder_seq_len,dtype=tf.float32)
    
    return encoder_outputs, encoder_states
`
This function has 4 space in the starting of the comment. make other function comments also like this function

**Comment by vishal2612200** [comment]
@lucko515 you can close this now. It has been solved now.

**Comment by khritish29** [comment]
I am still getting indentation error can someone share the corrected file with me please
